surface_segmentation:
  # Scene cropping
  crop_min_x: 0
  crop_min_y: -0.5
  crop_min_z: 0.05
  crop_max_x: 1.3
  crop_max_y: 0.5
  crop_max_z: 2.0
  
  # Plane detection
  horizontal_tolerance_degrees: 10.0
  margin_above_surface: 0.025
  # Multi-plane detection
  min_surface_size: 5000
  max_point_distance: 0.015
  
  # Clustering
  cluster_distance: 0.01
  min_cluster_size: 100
  max_cluster_size: 5000 #10000
  
  # For downsampling saved point clouds
  vox_leaf_size: 0.01
